docs/Using-Virtual-Environment.md

```diff
@@ -0,0 +1,91 @@
+# Using Virtual Environments for Animal-AI
```

Comment:
Same comment for this one as the jupyter notebooks one. We don't want to commit to maintaining documentation on python venvs when we don't need to. I think a note that these things are useful and link to some external documentation can be handy though.

Reply:
We need to acknowledge that probably most of our users may be very inexperienced with programming and related in general. Having such docs will keep our docs organised and sufficient in their integratiion, where relevant.